Source/Blazorise/Components/Droppable/DropZone.razor

```diff
@@ -5,18 +5,51 @@
     <div @ref="@ElementRef" id="@ElementId" class="@ClassNames" style="@StyleNames" @ondragenter="@OnDragEnterHandler" @ondragleave="@OnDragLeaveHandler" @ondrop="@OnDropHandler" @attributes="@Attributes">
         @ChildContent
 
-        @foreach ( var item in GetItems() )
+        @if ( OnlyZone != true )
         {
-            <_Draggable @key="@item" TItem="TItem" Item="@item" ZoneName="@Name" DraggingClass="@GetItemDraggingClass()" Disabled="@GetItemDisabled(item)" DisabledClass="@GetItemDisabledClass()" DragStarted="@OnDragStarted" DragEnded="@OnDragEnded">
-                @{
-                    var renderer = ItemTemplate ?? ParentContainer?.ItemTemplate;
+            int index = 0;
+            var transactionIndex = ParentContainer?.GetTransactionIndex() ?? -1;
+            var items = GetItems();
+
+            @if ( AllowReorder )
+            {
+                @if ( items.Any() == false )
```

Comment:
Same thing as before do we prefer this `items.Any() == false` over `!items.Any()` ?